scripts/validate-metadata.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# PACKAGE METADATA
+# ----------------
+if [[ $(python setup.py --name) == "UNKNOWN" ]] ; then
+    echo "[err] metadata: name missing"
+    exit 2
+fi
+
+if [[ $(python setup.py --version) == "0.0.0" ]] ; then
+    echo "[err] metadata: version missing"
+    exit 2
+fi
+
+if ! python setup.py --version | grep -q -E '^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$' ; then
+    echo "[err] metadata: version is not according to versioning standards"
+    exit 2
+fi
+
+
```

Comment:
For consistency, please remove extra newline.

Reply:
Done